### PR TITLE
the media query for desktop is done and also css styling for desktop …

### PR DESCRIPTION
**Pull request by MMhaidari**
# What is added new 
- desktop version support added
- media queries added for desktop support
- CSS styling is finished for desktop support

### CODE REVIEW

**Comment by MMhaidari** [comment]
# What is added new 
- desktop version support added
- media queries added for desktop support
- CSS styling is finished for desktop support

File: index.html [hunk 1]
```diff
@@ -248,16 +268,15 @@ <h1 class="form-header">Contact me</h1>
...
-          method="POST">
+          method="POST" class="form-elements">
             <input type="text" name="user_name" placeholder="Yeremias" maxlength="30"/>
             <input type="email" name="user_email" placeholder="natajayanj14@gmail.com" />
             <textarea name="user_text" id="msg" cols="30" rows="10" placeholder="Write your message here..." maxlength="500"></textarea>
-        </div>
-        <button class="btn-submit">Get It Touch</button>
-        </form>
+            <button class="btn-submit">Get It Touch</button>
+          </form>
```
Comment (Ibizugbe) [inline index.html:277]: Your contact and about section on the desktop looks different from the design, it is a requirement that you stick to the design for this milestone🤗😉

your app
your contact desktop

design
design about desktop

kindly make comparison between your project and the design and make the required changes🤗

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @MMhaidari 👋, 

68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!🤗

## Highlights✅
- Project is professionally documented👍
- Linters are passing👍


## Required Changes ♻️
- I recommend you stick to the design as you make your application responsive👍 you can always cross check the mobile and desktop views on the design while you buold your project. to avoid deviations from the design🤗
- Your validations for your forms are missing, I think you can bring them back again😉 add `required` and `max-length` properties as stated in the previous milestones🤗


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MMhaidari** [comment]
# what is added new 
- conflicts problems solved while changing desktop support to mobile support and via versa
- changes added to cards and also background color
- changes added to the contact form as instructed

**Review by frank1738 — APPROVED** [review]
Hi @MMhaidari , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**